src/main/fc/rc_modes.h

```diff
@@ -103,6 +103,8 @@ typedef struct boxBitmask_s { uint32_t bits[(CHECKBOX_ITEM_COUNT + 31) / 32]; }
 #define MIN_MODE_RANGE_STEP 0
 #define MAX_MODE_RANGE_STEP ((CHANNEL_RANGE_MAX - CHANNEL_RANGE_MIN) / 25)
 
+#define STICKY_MODE_BOOT_DELAY 5e6
```

Comment:
Make this `STICKY_MODE_BOOT_DELAY_US` to indicate the units. Also, if it's not intended to be used outside of `rc_modes.c` (i.e. not part of the interface of the unit), it should be defined in there.

Reply:
👍 to both